### PR TITLE
160088425  Signup/Registration UI design with materialize 

### PR DESCRIPTION
**Pull request by migot01**
#### What does this PR do?
Design signup ui with the use of materialize.

#### Description of tasks to be completed
Implement designs for the Signup/Registration Page.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ft-revisit-signup-ui-design-160088425
```
```sh
$ cd ah-titans-frontend
```

>Install the dependencies using the command below.

```sh
$ npm install
```
>Run the application as below.
```sh
$ npm start
```
>Now you can navigate to your browser and you should be able to see a page as the one shown on the screenshot below.

image

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160088425](https://www.pivotaltracker.com/story/show/160088425)

### CODE REVIEW

**Comment by dcDalin** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:
  - `src/App/__tests__/index.js`
- `src/Signup/__tests__/Signup.test.js`
- `src/Signup/components/__tests__/SignupForm.js`
  If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by janetnim — APPROVED** [review]

**Review by ronaldndirangu — APPROVED** [review]
Great work guys. I think this PR is ready to be merged.

**Comment by jmnyarega** [comment]
Kindy add more tests @migot01

File: src/App/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import Header from '../Header/components';
+// import Header from '../Header/components';
```
Comment (jmnyarega) [inline src/App/index.js]: Please avoid pushing commented code

File: src/App/index.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 import Main from '../Main/components';
 
 const App = () => (
   <div>
-    <Header />
+    {/* <Header /> */}
```
Comment (jmnyarega) [inline src/App/index.js]: Same here, don't push commented code

**Review by jmnyarega — CHANGES_REQUESTED** [review]

File: src/Signup/Signup.test.js [hunk 3]
```diff
@@ -7,13 +7,19 @@ import SignupForm from './components/SignupForm';
...
+    const signupFormWrapper = shallow(<SignupForm />);
+    expect(signupFormWrapper.find('Input').length).toEqual(4);
   });
 });
 
-describe('Signup form', () => {
-  it('returns a form', () => {
+describe('Signup button', () => {
+  it('renders a signup button', () => {
     const signupFormWrapper = shallow(<SignupForm />);
```
Comment (jmnyarega) [inline src/Signup/Signup.test.js]: Can we have this wrapper declared only once? Looks like code repetition. @migot01

File: src/Signup/Signup.test.js [hunk 4]
```diff
@@ -7,13 +7,19 @@ import SignupForm from './components/SignupForm';
 describe('User Signup', () => {
   it('renders the signup form succefully', () => {
     shallow(<Signup />);
+  });
+});
 
+describe('Signup input fields', () => {
+  it('renders input fields', () => {
+    const signupFormWrapper = shallow(<SignupForm />);
```
Comment (jmnyarega) [inline src/Signup/Signup.test.js]: Repeated code !!!

File: src/Signup/components/SignupForm.js [hunk 5]
```diff
@@ -1,43 +1,43 @@
...
+            </Col>
+          </Row>
+        </Card>
+      </Col>
+    </Row>
   </div>
 );
 
+SignupForm.prototype = {
+  submit: PropTypes.func.isRequired,
```
Comment (jmnyarega) [inline src/Signup/components/SignupForm.js]: This should be `SignupForm.propTypes` not `SignupForm.prototype`, you have defined `submit` which you are not using, just create an empty `SignupForm.propTypes` object i.e
```
SignupForm.prototype = {};
```


👇 
[React docs on proptypes](https://reactjs.org/docs/typechecking-with-proptypes.html)

**Pull request merged**